E/Elemental/build_tarballs.jl

```diff
@@ -67,9 +86,20 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("METIS_jll"),
-    Dependency("MPICH_jll"),
     Dependency("OpenBLAS_jll"),
 ]
 
+platforms, platform_dependencies = MPI.augment_platforms(platforms)
+# Avoid platforms where the MPI implementation isn't supported
+# AMReX's cmake stage fails with OpenMPI on almost all architectures
+platforms = filter(p -> !(p["mpi"] == "openmpi"), platforms)
+# With MPItrampoline, select only those platforms where MPItrampoline is actually built
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && (Sys.iswindows(p) || libc(p) == "musl")), platforms)
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
+append!(dependencies, platform_dependencies)
+
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+# GCC 4 does not support -std=c++14
+# GCC 5 does not support the OpenMP function `omp_get_num_places`
```

Comment:
Yeah, openmp is subtly broken on PowerPC with gcc 4 and 5.  I've never got whether that's an upstream issue, or of our toolchain.